sql/core/src/test/resources/sql-tests/results/typeCoercion/native/implicitTypeCasts.sql.out

```diff
@@ -317,38 +317,38 @@ struct<second(CAST(10:11:12 AS TIMESTAMP)):int>
 -- !query 39
 select 1 like '%' FROM t
 -- !query 39 schema
-struct<CAST(1 AS STRING) LIKE %:boolean>
+struct<CAST(1 AS STRING) LIKE % \:boolean>
```

Comment:
Could you remove this change by overriding some functions, e.g., simpleString?

Reply:
Yes, I can discard overriding the function `sql`.